langchain/schema.py

```diff
@@ -353,21 +460,54 @@ def clear(self) -> None:
 
 
 class Document(Serializable):
-    """Interface for interacting with a document."""
+    """Class for storing a piece of text and associated metadata."""
 
     page_content: str
+    """String text."""
```

Comment:
lets beef up this description

Reply:
not really sure what more to put given how generic idea of a Document is